source/Stargate-Examples-Tests/PetStoreAPITest.class.st

```diff
@@ -125,7 +273,7 @@ PetStoreAPITest >> testCreatePet [
 
 	self
 		assert: response isCreated;
-		assert: response location equals: 'http://localhost:9999/pets/1';
+		assert: response location equals: ( self baseUrl / 'pets' / '1' ) asString;
```

Comment:
In other places we have  `assertUrl:equals:` that compares both values turning them into URLs first

Reply:
hmm the CORS headers? I added assertUrl:equals: as an utility method in other tests. You're suggesting I should add it to this tests?